src/pages/Home/sections/Home/Blogs.jsx

```diff
@@ -0,0 +1,361 @@
+                        <path strokeLinecap="round" strokeLinejoin="round" d="M15.75 19.5 8.25 12l7.5-7.5" />
+                    </svg>
+                </button> */}
+                    <button className='prev lg:-mt-12 lg:relative  lg:top-[50%] lg:left-[0%] lg:translate-y-[-50%] lg:p-3 w-fit h-fit md:p-3  bg-white rounded-full borderLine shadow md:absolute md:top-1/2 md:translate-y-[-50%] md:left-[2%] z-10  top-[39.5%] translate-y-[-50%]  absolute left-[-5%]  p-3   md:block' style={{ opacity: showPrevButton ? 1 : 0 }}>
+                    <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor" className="text-[#2D89C8] w-[24px] h-[24px] font-bold">
+                        <path strokeLinecap="round" strokeLinejoin="round" d="M15.75 19.5 8.25 12l7.5-7.5" />
+                    </svg>
+                </button>
+                <Swiper
+                    slidesPerView={"auto"}
+                    loop={true}
+                    speed={400}
+                   
+                    breakpoints={{
+                        768 : {
+                      spaceBetween : 20,
+                      slidesPerView: 2,
+                      initialSlide : 2,
+                      centeredSlides : true
+                        },
+                        300 : {
+                       spaceBetween : 20,
+                      slidesPerView: 1,
+                    //   initialSlide : 2,
+                      centeredSlides : true
+                        },
+                        1100 : {
+                          spaceBetween : 20,
+                          slidesPerView: 3,
+                          // initialSlide : 0,
+                          // centeredSlides : true
+                        }
+                      }}
+                    navigation={{
+                        nextEl: '.next',
+                        prevEl: '.prev',
+                        clickable: true,
+                    }}
+                    // pagination={{
+                    //   clickable: true,
+                    // }}
+                    onSlideChange={(swiper) => handleSlideChange(swiper)}
+                    modules={[Navigation]}
+                    className="blog_crousal"
+                >
+                    {/* <div className='flex gap-6'> */}
+                    {blogs.map((ele,idx)=>{
+                        
+                        return ( <SwiperSlide key={idx} className='w-[300px]' >
+                         <BlogCard blog={ele} imgUrl={thumbBlog[idx]} />
+                       </SwiperSlide>)
+                        // <BlogCard blog={ele} />
+                    })}
+                    {/* <SwiperSlide></SwiperSlide> */}
+                </Swiper>
+                {/* </div> */}
+                
+                {/* yh swipper ka right button ko hide krna ka code hai style={{ display: showNextButton ? "block" : "none" }} */}
```

Comment:
Kindly remove any comments that might not have future use - if to be used in the future kindly prepend with `//Future: ...comment`